src/components/show-edit-pod/show-edit-pod.test.js

```diff
@@ -0,0 +1,323 @@
+import * as React from "react";
+import ShowEditPod from "./show-edit-pod.component";
+import Textbox from "../textbox/textbox.component";
+import Content from "../content/content.component";
+import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
+
+import {
+  showEditPod,
+  ContentShowEditPod,
+  ClassShowEditPod,
+  ContentFormShowEditPod,
+  HideDeleteButtonShowEditPod,
+  editFieldsShowEditPod,
+  FormFooterShowEditPod,
+  CancelButtonShowEditPod,
+  SaveButtonShowEditPod,
+  DeleteButtonShowEditPod,
+  TitleShowEditPod,
+  EditContainerShowEditPod,
+  undoShowEditPod,
+  transitionNameShowEditPod,
+} from "../../../cypress/locators/show-edit-pod";
+
+const specialCharacters = ["mp150ú¿¡üßä", "!@#$%^*()_+-=~[];:.,?{}&\"'<>"];
+
+const ShowEditPodComponent = ({ ...props }) => {
+  return (
+    <ShowEditPod
+      title="Title"
+      onEdit={() => {}}
+      onDelete={() => {}}
+      saveText="Save"
+      deleteText="Delete"
+      cancelText="Cancel"
+      editFields={<Textbox label="Field" labelInline labelAlign="right" />}
+      {...props}
+    />
+  );
+};
+
+const UndoShowEditPodComponent = ({ ...props }) => {
```

Comment:
is there a possibility to combine this one with the above custom component?

Reply:
We don't want to use undo button for other tests as it's not common so I've created a new Component and position is same for edit and undo button so its better to keep it separate.